simulatedDevice.py

```diff
@@ -0,0 +1,185 @@
+import asyncio
+import random
+import RPi.GPIO as GPIO
+import datetime
+import time
+import uuid
+from azure.iot.device.aio import IoTHubDeviceClient
+from azure.iot.device import Message
+
+movement_counter = 0
+doorstate = "closed"
+roomstate = "vacant"
+timeMovementCounterStarted = datetime.datetime.now()
+dateLastSeenMovement = datetime.datetime.now()
+
+pin_motion = 5
+pin_door = 6
+
+GPIO.setmode(GPIO.BCM)
+GPIO.setup(pin_motion,GPIO.IN)
+GPIO.setup(pin_door, GPIO.IN)
+
+
+#def sendTelemetryData():
+    
+async def checkDoorState(device_client):
+    #forever asynchronously check:
+    # get the reading of the gpio pin that refers to the reed switch
+    # if that reading is different to the current state of the door
+    # change the current state of the door and call the doorchangedstate method
+    #globals
+    global doorstate
+    print("door State Running")
+    while True:
+        pin_door_state = GPIO.input(pin_door)
+        temp_door_state = "closed"
+        if pin_door_state != 1:
+            temp_door_state = "opened"
+        
+        if temp_door_state != doorstate:
+            doorstate = temp_door_state
+            await doorChangedState(device_client)
+        await asyncio.sleep(0.01)
+            
+            
+        
+    
+async def isSomeoneInTheRoom(device_client):
+      #take readings from the motion sensor every 0.02 seconds to determine movement
+        # if movement happens start a movement counter which resets every 30 seconds
+        # when the movement counter reaches x mark the room as occupied
+        
+        #globals
+    global roomstate
+    global movement_counter
+    global timeMovementCounterStarted
+    global dateLastSeenMovement
+    global doorstate
+    print("room checker running")
+    while True:
+        motion_detected = GPIO.input(pin_motion)
+        if(motion_detected == 1):
```

Comment:
Parenthesis here are inconsitent with the rest of the if statements (no parenthesis)